app/src/main/java/com/brandon/forzenbook/view/composables/CreateUserComposables.kt

```diff
@@ -46,104 +62,239 @@ fun CreateAccountContent(
+                )
+                Text(
+                    modifier = Modifier
+                        .padding(LocalDimens.current.paddingExtraSmall)
+                        .weight(CREATE_ACCOUNT_TITLE_WEIGHT),
+                    text = resources.getString(R.string.createAccountTitleText),
+                    style = MaterialTheme.typography.titleLarge,
+                    color = Color.Black,
+                    maxLines = TEXT_FIELD_MAX_LINES,
+                    overflow = TextOverflow.Ellipsis,
+                )
+            }
+        },
+        content = { padding ->
+            YellowColumn(
+                modifier = Modifier.padding(padding)
+            ) {
+                InputInfoTextField(
+                    hint = resources.getString(R.string.createAccountFirstNameHint),
+                    currentText = firstName,
+                    onTextChange = { firstName = it },
+                )
+                InputInfoTextField(
+                    hint = resources.getString(R.string.createAccountLastNameHint),
+                    currentText = lastName,
+                    onTextChange = { lastName = it },
+                )
+                ReadOnlyDateTextField(
+                    hint = resources.getString(R.string.createAccountBirthDateFormatHint),
+                    currentText = dateOfBirth,
+                    onTextChange = { dateOfBirth = it },
+                    characterLimit = DATE_OF_BIRTH_CHAR_LIMIT,
+                    onClickEvent = { dialog.show() }
+                )
+                if (state.isDateError) TextFieldErrorText(text = resources.getString(R.string.createAccountDateError))
+                InputInfoTextField(
+                    hint = resources.getString(R.string.createAccountEmailHint),
+                    onTextChange = { email = it },
+                    currentText = email,
+                    characterLimit = EMAIL_CHAR_LIMIT,
+                    onMaxCharacterLength = { emailError = it },
+                )
+                if (state.isEmailError) TextFieldErrorText(text = resources.getString(R.string.createAccountEmailError))
+                if (state.isDuplicateUser) TextFieldErrorText(text = resources.getString(R.string.createAccountDuplicateUserError))
+                InputInfoTextField(
+                    hint = resources.getString(R.string.createAccountLocationHint),
+                    onTextChange = { location = it },
+                    currentText = location,
+                    onMaxCharacterLength = { locationError = it },
+                    keyboardOptions = KeyboardOptions(imeAction = ImeAction.Go)
+                )
+                if (state.isLocationError) TextFieldErrorText(text = resources.getString(R.string.createAccountLocationError))
+                BlackButton(text = resources.getString(R.string.createAccountCreateButtonText)) {
+                    keyboardController?.hide()
```

Comment:
I believe this can throw. Wrap it

Reply:
have this wrapped with an verbose log in case it ever happens, so it can be tracked. Will remove logging if its not needed, since there really isn't a reason to catch anything here for the moment, but it could be useful if it needs to be debugged later.